### PR TITLE
feat: Add module version to the /status output

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Addresses https://github.com/appium/appium/issues/19921

### CODE REVIEW

File: WebDriverAgentLib/Commands/FBSessionCommands.m [hunk 1]
```diff
@@ -279,6 +279,11 @@ + (NSArray *)routes
   if (nil != upgradeTimestamp && upgradeTimestamp.length > 0) {
     [buildInfo setObject:upgradeTimestamp forKey:@"upgradedAt"];
   }
+  NSDictionary *infoDict = [[NSBundle bundleForClass:self.class] infoDictionary];
+  NSString *version = [infoDict objectForKey:@"CFBundleShortVersionString"];
```
Comment (KazuCocoa) [inline WebDriverAgentLib/Commands/FBSessionCommands.m:283]: Let me double check again later. xcodebuild could override this plist as `1` in its build time. Xcode did before for `CFBundleVersion` when I directly changed the info.plist over Xcode. Maybe I haven't checked this `CFBundleShortVersionString`

Oh, maybe i haven't checked `WebDriverAgentLib`'s update

**Comment by youngfreeFJS** [comment]
woooooo ! 👍

File: WebDriverAgentLib/Commands/FBSessionCommands.m (hunk 1 above)
Comment (KazuCocoa) [reply WebDriverAgentLib/Commands/FBSessionCommands.m:283]:
```
diff --git a/WebDriverAgentLib/Info.plist b/WebDriverAgentLib/Info.plist
index 1b89f4fc..15bcde45 100644
--- a/WebDriverAgentLib/Info.plist
+++ b/WebDriverAgentLib/Info.plist
@@ -15,11 +15,11 @@
        <key>CFBundlePackageType</key>
        <string>FMWK</string>
        <key>CFBundleShortVersionString</key>
-       <string>1.0</string>
+       <string>2.0.0</string>
        <key>CFBundleSignature</key>
        <string>????</string>
        <key>CFBundleVersion</key>
-       <string>$(CURRENT_PROJECT_VERSION)</string>
+       <string>2.0.0</string>
        <key>NSPrincipalClass</key>
        <string></string>
 </dict>
```

Info.plist after ` xcodebuild clean build-for-testing -project WebDriverAgent.xcodeproj -derivedDataPath ./build_wda -scheme WebDriverAgentRunner -destination generic/platform=iOS CODE_SIGNING_ALLOWED=NO ARCHS=arm64` in `WebDriverAgentRunner-Runner.app`:

The `CFBundleShortVersionString` and `CFBundleVersion` were 1. 

```
        <key>CFBundleAllowMixedLocalizations</key>
        <true/>
        <key>CFBundleDevelopmentRegion</key>
        <string>en</string>
        <key>CFBundleExecutable</key>
        <string>WebDriverAgentRunner-Runner</string>
        <key>CFBundleIdentifier</key>
        <string>com.facebook.WebDriverAgentRunner.xctrunner</string>
        <key>CFBundleInfoDictionaryVersion</key>
        <string>6.0</string>
        <key>CFBundleName</key>
        <string>WebDriverAgentRunner-Runner</string>
        <key>CFBundlePackageType</key>
        <string>APPL</string>
        <key>CFBundleShortVersionString</key>
        <string>1.0</string>
        <key>CFBundleSignature</key>
        <string>????</string>
        <key>CFBundleSupportedPlatforms</key>
        <array>
                <string>iPhoneOS</string>
        </array>
        <key>CFBundleVersion</key>
        <string>1</string>
        <key>DTCompiler</key>
        <string>com.apple.compilers.llvm.clang.1_0</string>
        <key>DTPlatformBuild</key>
        <string>21A289</string>
        <key>DTPlatformName</key>
        <string>iphoneos</string>
        <key>DTPlatformVersion</key>
        <string>17.0</string>
```

So, it looks like still the version in `info.plist` before building was overridden as 1/1.0. (as https://github.com/appium/appium/issues/19921#issuecomment-2017237466 )

Did your local env work expectedly? @mykola-mokhnach 

I haven't checked with this code itself. let me double check later

Comment (KazuCocoa) [reply WebDriverAgentLib/Commands/FBSessionCommands.m:283]: oh, nice. I found the info.plist in:

```
Build/Products/Debug-iphoneos/WebDriverAgentRunner-Runner.app/PlugIns/WebDriverAgentRunner.xctest/Frameworks/WebDriverAgentLib.framework
```

only the info.plist keeps this value. The top plist doesn't keep the value but the `info.plist` in the WebDriverAgentLib as embedded framework could.

I only investigated the top one, usually used as an app.

**Review by KazuCocoa — APPROVED** [review]

File: WebDriverAgentLib/Commands/FBSessionCommands.m (hunk 1 above)
Comment (mykola-mokhnach) [reply WebDriverAgentLib/Commands/FBSessionCommands.m:283]: yes, we get the value from the framework's plist. The one for the main runner gets overridden by Xcode as you mentioned above

**Pull request merged**